src/main/java/io/stargate/sgv2/jsonapi/service/resolver/sort/TableCqlSortClauseResolver.java

```diff
@@ -348,7 +352,7 @@ private WithWarnings<OrderByCqlClause> resolveVectorSort(
         new TableOrderByANNCqlClause(
             vectorSortColumn,
             cqlVector,
-            commandContext.getConfig(OperationsConfig.class).maxVectorSearchLimit()),
+            commandContext.config().get(OperationsConfig.class).maxVectorSearchLimit()),
```

Comment:
idea/question: why not have "CommandContext.getConfig()" as before, as convenience method?

Reply:
because I like making things harder, leaving open to change in the next PR